test/schema-validation.test.js

```diff
@@ -15,9 +15,13 @@ describe('json schema validation', () => {
     }).doesNotThrow;
   });
 
-  it('throws if an incorrect config is passed in', () => {
```

Comment:
I'm sry about this 😅 I should have taken a look sooner but could you add `process.env.JEST = true` on L3-4 instead so `schema-utils` will rethrow the `{Error}` instead of exiting while running tests ? The 'old' test should work then. 

```js
const err = () => new ExtractTextPlugin({ filename: 1 })

expect(err).toThrow()
// Test if we get the correct err.message
expect(err).toThrowErrorMatchingSnapshot()
```

Reply:
But in reality, doesn't it write to console and exit, not throw? My test was written to match that scenario. 

The test you describe would essentially be "when in testing with a bad config, does it throw?" Unless I'm missing something, that doesn't seem like a very valuable test.

(Not sure why one of the checks is failing, I've been on holiday and but tests were passing on my computer before I left. Can look into this tomorrow, and will update based on your response.)

Reply:
The test should be changed back, but please hold on for ~1-2 days we resolved this in [webpack core (#6035)](https://github.com/webpack/webpack/pull/6035) and [`schema-utils` (#17)](https://github.com/webpack-contrib/schema-utils/pull/17)

Reply:
@michael-ciniawsky Can you explain why the test should be changed back? I am still confused about what it is testing.

Reply:
It's tests for malformed options based on a JSON Schema and should work without the need to mock anything in the test, please revert the test changes and I will finally land this on `master` :)